src/pages/ChooseArea.svelte

```diff
@@ -89,9 +92,50 @@
         window.location.href = `scheme.html?authority=${name}`;
       });
     });
+  });
+
+  function loadFile(e: Event) {
```

Comment:
Can we share this in the `FileInput.svelte`? feels like a lot of this logic must be shared between the two locations

Reply:
Done. It's unfortunate that `FileReader` uses callback APIs and not async/await